test/orbit_web/controllers/sign_in_controller_test.exs

```diff
@@ -200,5 +214,48 @@ defmodule OrbitWeb.SignInControllerTest do
 
       assert "Employee not found" = response(conn, 404)
     end
+
+    @tag :authenticated
+    test "replies 400 when invalid override (invalid type)", %{conn: conn} do
+      conn =
+        post(conn, ~p"/api/signin", %{
+          "signed_in_employee_badge" => "123",
+          "signed_in_at" => 1_721_164_459,
+          "line" => "blue",
+          "method" => "manual",
+          "radio_number" => "22",
+          "override" => [
+            %{
+              "expires" => "2024-06-08T00:00:00.000-04:00",
+              "rail_line" => "blue",
+              "type" => "NOT_A_REAL_TYPE"
+            }
+          ]
+        })
+
+      assert "Invalid override\n" = response(conn, 400)
+    end
+
+    @tag :authenticated
+    test "replies 400 when invalid override (extraneous key)", %{conn: conn} do
```

Comment:
Why is this important to verify in a test? Do we care if the server ignores vs 400s on an extra key?

Reply:
It seemed like extraneous keys shouldn't be allowed, since that suggests there's something amiss. It's like homebrew reverse zod. Although I admit that might be overly strict and prone to issues. IDK, does that change your leaning?